maraboupy/MarabouCore.cpp

```diff
@@ -103,7 +103,9 @@ void createInputQuery(InputQuery &inputQuery, std::string networkFilePath, std::
     printf( "Property: None\n" );
 }
 
-std::pair<std::map<int, double>, Statistics> solve(InputQuery inputQuery, std::string redirect="", unsigned timeout=0){
+/* The default parameters here are just for readability, you should specify
+ * them in the to make them work*/
```

Comment:
Sentence is broken

Reply:
That is to keep it less then 80 characters.
We don't want that?